// _@vuepress_core@1.5.2@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/apple/fish-wheels/node_modules/_@vuepress_core@1.5.2@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-9982b756",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-9982b756").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-0513ba15",
    path: "/API%E8%AE%BE%E8%AE%A1%E5%8E%9F%E5%88%99/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0513ba15").then(next)
    },
  },
  {
    path: "/API设计原则/",
    redirect: "/API%E8%AE%BE%E8%AE%A1%E5%8E%9F%E5%88%99/"
  },
  {
    path: "/API%E8%AE%BE%E8%AE%A1%E5%8E%9F%E5%88%99/index.html",
    redirect: "/API%E8%AE%BE%E8%AE%A1%E5%8E%9F%E5%88%99/"
  },
  {
    name: "v-bcc3d8f4",
    path: "/components/button.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-bcc3d8f4").then(next)
    },
  },
  {
    name: "v-26ab5e46",
    path: "/components/calendar.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-26ab5e46").then(next)
    },
  },
  {
    name: "v-64e6fe46",
    path: "/components/cascader.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-64e6fe46").then(next)
    },
  },
  {
    name: "v-4dc65f26",
    path: "/components/collapse.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4dc65f26").then(next)
    },
  },
  {
    name: "v-2512c6c6",
    path: "/components/grid.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2512c6c6").then(next)
    },
  },
  {
    name: "v-40bce5a6",
    path: "/components/icon.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-40bce5a6").then(next)
    },
  },
  {
    name: "v-7a613270",
    path: "/components/input.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7a613270").then(next)
    },
  },
  {
    name: "v-f8825f84",
    path: "/components/pager.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-f8825f84").then(next)
    },
  },
  {
    name: "v-78504d0c",
    path: "/components/popover.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-78504d0c").then(next)
    },
  },
  {
    name: "v-71c5e586",
    path: "/components/slides.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-71c5e586").then(next)
    },
  },
  {
    name: "v-e7f7fc74",
    path: "/components/tabs.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-e7f7fc74").then(next)
    },
  },
  {
    name: "v-dbde9474",
    path: "/components/tree.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-dbde9474").then(next)
    },
  },
  {
    name: "v-f46bef42",
    path: "/get-started/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-f46bef42").then(next)
    },
  },
  {
    path: "/get-started/index.html",
    redirect: "/get-started/"
  },
  {
    name: "v-5bba2bb5",
    path: "/introduce/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5bba2bb5").then(next)
    },
  },
  {
    path: "/introduce/index.html",
    redirect: "/introduce/"
  },
  {
    name: "v-48bb193d",
    path: "/install/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-48bb193d").then(next)
    },
  },
  {
    path: "/install/index.html",
    redirect: "/install/"
  },
  {
    name: "v-d557027e",
    path: "/vue-cli/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-d557027e").then(next)
    },
  },
  {
    path: "/vue-cli/index.html",
    redirect: "/vue-cli/"
  },
  {
    name: "v-17e57352",
    path: "/%E5%8D%95%E5%85%83%E6%B5%8B%E8%AF%95%E6%80%BB%E7%BB%93/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-17e57352").then(next)
    },
  },
  {
    path: "/单元测试总结/",
    redirect: "/%E5%8D%95%E5%85%83%E6%B5%8B%E8%AF%95%E6%80%BB%E7%BB%93/"
  },
  {
    path: "/%E5%8D%95%E5%85%83%E6%B5%8B%E8%AF%95%E6%80%BB%E7%BB%93/index.html",
    redirect: "/%E5%8D%95%E5%85%83%E6%B5%8B%E8%AF%95%E6%80%BB%E7%BB%93/"
  },
  {
    name: "v-4bc0a0b5",
    path: "/vue-summary/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4bc0a0b5").then(next)
    },
  },
  {
    path: "/vue-summary/index.html",
    redirect: "/vue-summary/"
  },
  {
    name: "v-5fd721d0",
    path: "/%E5%BC%80%E5%8F%91%E8%A7%84%E8%8C%83/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5fd721d0").then(next)
    },
  },
  {
    path: "/开发规范/",
    redirect: "/%E5%BC%80%E5%8F%91%E8%A7%84%E8%8C%83/"
  },
  {
    path: "/%E5%BC%80%E5%8F%91%E8%A7%84%E8%8C%83/index.html",
    redirect: "/%E5%BC%80%E5%8F%91%E8%A7%84%E8%8C%83/"
  },
  {
    name: "v-ca0d7a02",
    path: "/%E5%BC%80%E5%8F%91%E6%B5%81%E7%A8%8B/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-ca0d7a02").then(next)
    },
  },
  {
    path: "/开发流程/",
    redirect: "/%E5%BC%80%E5%8F%91%E6%B5%81%E7%A8%8B/"
  },
  {
    path: "/%E5%BC%80%E5%8F%91%E6%B5%81%E7%A8%8B/index.html",
    redirect: "/%E5%BC%80%E5%8F%91%E6%B5%81%E7%A8%8B/"
  },
  {
    name: "v-246d4f06",
    path: "/%E7%BB%84%E4%BB%B6%E9%9C%80%E6%B1%82%E5%88%86%E6%9E%90/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-246d4f06").then(next)
    },
  },
  {
    path: "/组件需求分析/",
    redirect: "/%E7%BB%84%E4%BB%B6%E9%9C%80%E6%B1%82%E5%88%86%E6%9E%90/"
  },
  {
    path: "/%E7%BB%84%E4%BB%B6%E9%9C%80%E6%B1%82%E5%88%86%E6%9E%90/index.html",
    redirect: "/%E7%BB%84%E4%BB%B6%E9%9C%80%E6%B1%82%E5%88%86%E6%9E%90/"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
